tsconfig.json

```diff
@@ -9,12 +9,11 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "sourceMap": true,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "typeRoots": ["./types", "./node_modules/@types"]
   },
   "include": [
     "src/**/*"
   ],
-  "exclude": [
-    "node_modules"
-  ]
-}
\ No newline at end of file
+  "exclude": ["node_modules", "types"]
```

Comment:
:question: can we revert this formatting change, not related to the subject (we already have a large amount of changes, so avoiding extra change keeps this PR more focussed)

Reply:
sure, plus it is part of next PR, removing